src/components/PressableWithSecondaryInteraction/index.native.tsx

```diff
@@ -36,7 +36,7 @@ function PressableWithSecondaryInteraction(
                 suppressHighlighting={suppressHighlighting}
                 onLongPress={onSecondaryInteraction ? executeSecondaryInteraction : undefined}
             >
-                {children}
+                {children as ReactNode}
```

Comment:
@blazejkustra please check if this is good enough.  This `<Text>` component doesn't expect to get `((state: PressableStateCallbackType) => React.ReactNode)` so I used mapping to `ReactNode | string`

Reply:
Seems alright for me, Błażej was the one migrating Pressables so let's wait for his opinion as well though 😄 cc @blazejkustra